src/ray/protobuf/ray_client.proto

```diff
@@ -256,17 +256,9 @@ message InitResponse {
   string msg = 2;
```

Comment:
Since you modified the Proto, can you update the protocol version here:
https://github.com/ray-project/ray/blob/34d3d9294c50aea4005b7367404f6a5d9e0c2698/python/ray/util/client/__init__.py#L11